src/olympia/versions/models.py

```diff
@@ -831,6 +831,22 @@ def was_auto_approved(self):
             pass
         return False
 
+    @property
+    def has_been_human_reviewed(self):
+        """Return whether or not this version was reviewed by a human."""
+        from olympia.reviewers.models import AutoApprovalSummary
+
+        autoapproval = AutoApprovalSummary.objects.filter(version=self).first()
+
+        return (
+            self.file.status == amo.STATUS_APPROVED
+            and (
+                not autoapproval
+                or autoapproval.verdict == amo.NOT_AUTO_APPROVED
+                or autoapproval.confirmed is True
+            )
+        ) or (self.file.status == amo.STATUS_DISABLED and self.file.reviewed)
```

Comment:
I believe `file.reviewed` does not work unlisted, unless that has changed recently?

Reply:
Both `reject_latest_version` (that only works with listed) and reject_multiple_versions (that's available to both channels, and used by the `auto_reject` command, the blocklist admin too) appear to call `set_file` that sets `file.reviewed`.

Any suggestions for improvements to these checks (i.e. to determine if a given version has been human reviewed or not) would be welcome though.